app/visualisation_methods/sensitivity/info.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Sensitivity design",
+  "id": "sensitivity",
+  "group": "params",
+  "image": "sensitivity_desing_icon.png",
+  "description":  "Shows results of sensitivity analysis with conducted specific method (automatic experiment type)."
```

Comment:
conducted with chyba powinno być, a nie odwrotnie? Jakoś niegramatycznie to brzmi..
Może takie coś: Shows results of sensitivity analysis if it was applied to this experiment using automatic experiment option. (?)

Jeśli macie inne propozycje - piszcie.